### DISCUSSION TITLE
两个雷达图并排，其中一个雷达图高度会无限放大

### DISCUSSION THREAD

**Discussion by whale-jung** [antvis/G2Plot#3649] [Q&A]
是需要配置什么参数吗

**Comment by whale-jung** [comment]
复现地址：https://riddle.alibaba-inc.com/riddles/c5a64000

**Reply by whale-jung** [reply]
这个注释需要取消下

<img width="511" alt="image" src="https://github.com/antvis/G2Plot/assets/80868414/39c42524-eda1-47b8-be4c-fa3b8b528123">

**Reply by hustcc** [reply]
应该图表会自动监听容器大小变化，然后重绘。检查下两个 div 是否会互相影响大小。
